lib/ccd/templates/problem_concern_act_condition_template.rb

```diff
@@ -10,10 +10,9 @@ def self.included(base)
       constraint 'mood_code', {:cardinality=>"1..1", :value=>"EVN"}
       
       # SHALL contain exactly one [1..1] templateId (CONF:16772) such that it
-      constraint 'template_id', {:cardinality=>"1..1"}
-      
       # SHALL contain exactly one [1..1] @root="2.16.840.1.113883.10.20.22.4.3" (CONF:16773).
-      constraint 'template_id.root', {:cardinality=>"1..1", :value=>"2.16.840.1.113883.10.20.22.4.3"}
+      constraint 'template_id', {:cardinality=>"1..1", :value=>{:root => "2.16.840.1.113883.10.20.22.4.3"}}
+      constraint 'template_id', {:cardinality=>"1..1", :value=>{:root => "2.16.840.1.113883.10.20.22.4.3", :extension => "2015-08-01"}}
```

Comment:
I'm guessing it's okay to hard-code the `extension` along with the `template_id`? Also, I'm guessing that they refer to a certain version of a CCDA?

Reply:
Just checked and it looks like it's okay to hard-code them. What are your thoughts on making these `ids` constants and storing them in a file dedicated to such constants?

Reply:
I'm scared of making too many changes in this gem because although we "own it now" I don't think we really want to & don't want to invest any more than necessary in it.  So I'd prefer not to start extracting constants if that's ok with you

Reply:
Cool!